### DISCUSSION TITLE
add projects to the comparisons page in documentation

### DISCUSSION THREAD

**Discussion by asnelling** [ory/kratos#1489] [Ideas]
**Is your feature request related to a problem? Please describe.**

In the documentation on the [comparison](https://www.ory.sh/kratos/docs/further-reading/comparison) page, no projects are listed with which to compare kratos.

**Describe the solution you'd like**

Add some comparable projects [here](https://github.com/ory/kratos/blob/master/docs/docs/further-reading/comparison.md).

**Additional context**

I'll try to demo at least one other solution and make a pull request with my findings. Hopefully others with experience can chime in.

**Comment by vinckr** [comment]
Hey, 
thanks for the reminder! I will convert this into a discussion.
Which projects would you like to see there?
